MrtCore/wpf_packaged_app/wpf_packaged_app/MainWindow.xaml.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Windows;
+using Microsoft.ApplicationModel.Resources;
+
+namespace wpf_packaged_app
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        public MainWindow(ResourceLoader resourceLoader, ResourceManager resourceManager)
+        {
+            m_resourceLoader = resourceLoader;
+            m_resourceManager = resourceManager;
+
+            // Create custom resource context for the window. Set the language to German.

```

Comment:
super nit I guess...

```suggestion
            // Create custom resource context for the window. Set the language to German (Germany).
```
Not sure if it matters. Feel free to won't fix. :)